ext/intl/uchar/uchar.stub.php

```diff
@@ -0,0 +1,338 @@
+<?php
+
+class IntlChar
+{
+    /**
+     * @param mixed $codepoint
```

Comment:
I'd use `int|string` for these.

Reply:
It's not strictly true in the zpp sense, but I think that's our general convention for using `@param` (rather than a proper type).